front/pages/api/v1/w/[wId]/assistant/conversations/[cId]/messages/index.ts

```diff
@@ -11,11 +15,12 @@ import { apiError, withLogging } from "@app/logger/withlogging";
 
 export type PostMessagesResponseBody = {
   message: UserMessageType;
+  agentMessages: AgentMessageType[];
```

Comment:
If we want to have the `resolveAfterFullGeneration` as a parameter yes, but why not keeping is as default? It makes the API more complex to use to have parameters to get different response structure. I'm in favor of keeping things simple and this route always answers with the agent messages.

Reply:
@PopDaph because the streaming use case does make sense and is the "industry standard" for LLM interaction as of today I think.
Also, we have external developers already relying on it today (Alan, our Slack bot, and probably a few more), so we need to keep the compatibility.
Would I be in favor of having this the default for the future? Most likely, but for now, we can't just break existing apps :/